### DISCUSSION TITLE
Coming New Features?

### DISCUSSION THREAD

**Discussion by axelcasauran** [serenity-is/Serenity#6314] [General]
Hello,

I'm just curious and excited, if there are coming new stuff for Serenity?
^_^

**Comment by volkanceylan** [comment]
Hmm what kind of new stuff are you expecting? Serenity is continuously getting updated for 10 years, watch the change log and commits. Keep in mind that it is a framework / platform not very visual thingy.

**Reply by axelcasauran** [reply]
Hello,

Actually, I'm more on expecting what is in the roadmap.. there are short / long term plan --to be specific the following:

1. OpenAuth / Social Login
2. Announcements Sample
3. Chat & Messaging System (with Signalr?)
4. Full inline editing functionality in grid
5. Mobile Application
